CHANGELOG.rst

```diff
@@ -85,6 +85,9 @@ Fixed
 * Update ``st2 execution get`` command to always include ``context.user``, ``start_timestamp`` and
   ``end_timestamp`` attributes. (improvement) #4739
 
+  Contributed by @stevemuskiewicz and @guzzijones
```

Comment:
This line needs to be moved not before, but after the actual changelog record to be part of the same block.

See how it looks like https://github.com/StackStorm/st2/blob/0fffbdf00cf4d547d254bb4319d6bb5fcb42e976/CHANGELOG.rst#fixed rendered.

Reply:
Wups, sorry about that.  I'll fix